### PR TITLE
Fixes #17803: Advise to not desinstall rudder server if possible

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17803

### CODE REVIEW

File: src/reference/modules/administration/pages/uninstall.adoc [hunk 1]
```diff
@@ -127,9 +127,15 @@ To delete a node from Rudder, you need to go to the node details page and click
 
 == Uninstalling Rudder server
 
+
 WARNING: Uninstalling the Rudder server package will remove configuration data,
-that will be lost if you don't have a backup. We advice to keep a backup of the server
-for future reference.
+that will be lost if you don't have a backup.
+
+In the general case, we advise to not uninstall Rudder server packages and to prefer to use a solution depending on your business and technical constraints where the underlying virtual machine is stopped, backupped, and hibernated.
```
Comment (amousset) [inline src/reference/modules/administration/pages/uninstall.adoc:134]: what is the point of advising not to uninstall the server? It makes not much sense but should not be a problem (and could be common for relyays that may have other services running).

Comment (fanf) [reply src/reference/modules/administration/pages/uninstall.adoc:134]: keeping configuration and data. There is an awfull lot of configuration details, history, event logs, etc that are in a rudder server, the kind of things that you are likely to need if you migrate to a new Rudder or even to a new config solution. Just having archives make exploitation of these data very hard.

Comment (fanf) [reply src/reference/modules/administration/pages/uninstall.adoc:134]: I was only talking about the rudder root server here, not relays. The title should be clearer.

**Comment by fanf** [comment]
PR updated with a new commit

File: src/reference/modules/administration/pages/uninstall.adoc [hunk 2]
```diff
@@ -125,11 +125,24 @@ Even if `rudder-agent` package is uninstalled from a node, the corresponding nod
 
 To delete a node from Rudder, you need to go to the node details page and click on the `Delete` button.
 
-== Uninstalling Rudder server
+== Uninstalling Rudder relay servers
+
+You can uninstall the Rudder packages with your package manager, and follow the
+same cleanup steps as for the agent.
+
+We advice to keep a backup a relay configuration files for future reference.
```
Comment (peckpeck) [inline src/reference/modules/administration/pages/uninstall.adoc:133]: "recommend" is better

**Pull request merged**